### PR TITLE
Fix crashes #16 #17

### PR DESCRIPTION
**Pull request by ashavit**

### CODE REVIEW

File: src/game/runners/ConsoleRunner.java [hunk 1]
```diff
@@ -178,15 +185,15 @@ private void playTurn() {
...
+            System.out.println("NICE JOB! you successfully hit a ship.\n");
         }
 
         this.showGameStatus();
     }
 
     private void showStatistics() {
         if (this.game == null) {
-            System.out.println("Statistics are available when game started.");
+            System.out.println("Statistics are available when game started.\n");
```
Comment (yuri1992) [inline src/game/runners/ConsoleRunner.java:196]: Why did you add all those "\n" ? println suppose to break line after the print. Does the cms did not print new lines after it ?

Comment (yuri1992) [reply src/game/runners/ConsoleRunner.java:196]: @ashavit

Comment (ashavit) [reply src/game/runners/ConsoleRunner.java:196]: @yuri1992 It does, I just wanted to add an empty line so it will be more clear when printing
